java/change-notes/2021-03-05-commons-lang-randomutils.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Added models for the Apache Commons-Lang `RandomUtils` class. This may lead to extra results from queries that check for proper use of random-number generators or those which check the range of possible random values that could be returned, including `java/improper-validation-of-array-index-code-specified` and `java/uncontrolled-arithmetic`.
```

Comment:
Really minor, but it looks like the project itself does not use a hyphen, see https://commons.apache.org/proper/commons-lang/
```suggestion
* Added models for the Apache Commons Lang `RandomUtils` class. This may lead to extra results from queries that check for proper use of random-number generators or those which check the range of possible random values that could be returned, including `java/improper-validation-of-array-index-code-specified` and `java/uncontrolled-arithmetic`.
```